serving/src/main/java/ai/djl/serving/http/ClusterRequestHandler.java

```diff
@@ -64,14 +66,40 @@ protected void handleRequest(
             QueryStringDecoder decoder,
             String[] segments)
             throws ModelException {
+        Path sshDir = Paths.get(System.getProperty("user.home")).resolve(".ssh");
+        Path authorizedKeysFilePath = sshDir.resolve("authorized_keys");
         switch (segments[2]) {
-            case "sshkey":
-                Path home = Paths.get(System.getProperty("user.home")).resolve(".ssh");
-                Path file = home.resolve("id_rsa.pub");
-                if (Files.notExists(file)) {
-                    sshkeygen(home.resolve("id_rsa").toString());
+            case "sshpublickey":
+                Path publicKeyFile = sshDir.resolve("id_rsa.pub");
+                if (Files.notExists(publicKeyFile)) {
+                    sshkeygen(sshDir.resolve("id_rsa").toString());
                 }
-                NettyUtils.sendFile(ctx, file, false);
+                try {
+                    Files.write(
```

Comment:
this can be simplified as:

```
Files.copy(publicKeyFile, authorizedKeysFilePath, StandardCopyOption.COPY_ATTRIBUTES);
```

Reply:
Removed this path as it does not seem to be necessary.